### PR TITLE
[14.0][MIG] project_stock_product_set: Migration to 14.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 14.0

Locked by:
- [x] https://github.com/OCA/project/pull/948

Please @CarlosRoca13 and @cesar-tecnativa can you review it?

@Tecnativa TT37029

### CODE REVIEW

**Comment by victoralmau** [comment]
Now, it's ready to review.

File: project_stock_product_set/security/ir.model.access.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_product_set_add_from_task_user,access_product_set_add_from_task_user,model_product_set_add_from_task,project.group_project_user,1,1,1,0
```
Comment (pedrobaeza) [inline project_stock_product_set/security/ir.model.access.csv]: Why not allowing to delete?

File: project_stock_product_set/security/ir.model.access.csv [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+access_product_set_add_from_task_manager,access_product_set_add_from_task_manager,model_product_set_add_from_task,project.group_project_manager,1,0,0,0
```
Comment (pedrobaeza) [inline project_stock_product_set/security/ir.model.access.csv]: This one is not needed at all

File: project_stock_product_set/i18n/es.po [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+#. module: project_stock_product_set
+#: model:ir.model.fields,field_description:project_stock_product_set.field_product_set_add_from_task__display_name
+msgid "Display Name"
+msgstr "Nombre mostrado"
+
+#. module: project_stock_product_set
+#: model:ir.model.fields,help:project_stock_product_set.field_product_set_add_from_task__skip_existing_products
+msgid "Enable this to not add new lines for products already included in SO lines."
+msgstr ""
```
Comment (cesar-tecnativa) [inline project_stock_product_set/i18n/es.po:61]: missing

File: project_stock_product_set/i18n/es.po [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+#. module: project_stock_product_set
+#: model:ir.model.fields,field_description:project_stock_product_set.field_product_set_add_from_task__task_id
+msgid "Task"
+msgstr "Tarea"
+
+#. module: project_stock_product_set
+#: model:ir.model,name:project_stock_product_set.model_product_set_add_from_task
+msgid "product.set.add.from.task"
+msgstr ""
```
Comment (cesar-tecnativa) [inline project_stock_product_set/i18n/es.po]: missing

**Review by cesar-tecnativa — CHANGES_REQUESTED** [review]

File: project_stock_product_set/i18n/es.po (hunk 3 above)
Comment (victoralmau) [reply project_stock_product_set/i18n/es.po:61]: Some terms do not need to be translated because they are not used and/or not shown.

**Comment by victoralmau** [comment]
Changes done.

File: project_stock_product_set/i18n/es.po (hunk 3 above)
Comment (pedrobaeza) [reply project_stock_product_set/i18n/es.po:61]: But for completion reasons, it's better to simply not let them untranslated

Comment (pedrobaeza) [reply project_stock_product_set/i18n/es.po:61]: And other times, we think they are not shown, but they are. Example: fields not shown on views, but shown on advanced search filters.

File: project_stock_product_set/wizard/product_set_add.xml [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+                    <button special="cancel" string="Cancel" class="oe_link" />
+                </footer>
+            </form>
+        </field>
+    </record>
+    <record
+        id="act_open_wizard_product_set_add_from_task"
+        model="ir.actions.act_window"
+    >
+        <field name="name">Add set in poject task</field>
```
Comment (pedrobaeza) [inline project_stock_product_set/wizard/product_set_add.xml]:
```suggestion
        <field name="name">Add a set in a project task</field>
```

**Review by cesar-tecnativa — APPROVED** [review]

File: project_stock_product_set/wizard/product_set_add.xml [hunk 6]
```diff
@@ -46,7 +45,7 @@
         id="act_open_wizard_product_set_add_from_task"
         model="ir.actions.act_window"
     >
-        <field name="name">Add set in poject task</field>
+        <field name="name">Add a set in a poject task</field>
```
Comment (pedrobaeza) [inline project_stock_product_set/wizard/product_set_add.xml]: s/poject/project

**Comment by victoralmau** [comment]
Changes done.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**